.github/workflows/publish.yml

```diff
@@ -21,7 +21,7 @@ jobs:
     strategy:
       matrix:
         # Support LTS versions based on https://nodejs.org/en/about/releases/
-        node-version: ['12', '14', '16']
```

Comment:
We probably no longer need to support anything below 18

Reply:
true